### ISSUE TITLE
Request to update Newtonsoft version to latest.

### ISSUE THREAD

**Issue by JamboBuenna** [RSuter/NSwag#1123]
Good morning. Just tried to pull in the NSwag into a new project I'm using. I noticed the version of Newtonsoft is hardcoded to 9.0.0.0. Preferably can you move this onto the latest version of Newtonsoft 10.0.3 or if not a supported version in NuGet, they seem to provide 9.0.1.

Hope this finds you well and appreciate any feedback or advice am new to C#

**Comment by RSuter** [comment]
If you also install a newer version of Newtonsoft.Json in your project, this new version should override the NSwag' specified version...

**Comment by JamboBuenna** [comment]
Hey Rico, Really appreciate you getting back to me. 

Strange, this didn't seem to happen for us, as said we're running 10.0.3 and are seeing ASP.net yellow screen exceptions NSwag not finding version 9 of Newtonsoft. (Apologies, can't share atm as not at dev pc).

**Comment by stijnherreman** [comment]
@JamboBuenna did you add the necessary binding redirect? The one below should work if you are using 10.0.3

```xml
<dependentAssembly>
  <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
  <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" />
</dependentAssembly>
```

**Comment by JamboBuenna** [comment]
Yes, thank you,  ended up doing something similar, the problem was for us that nuGet dependencies were caching old state somewhere in the build process for us, hence confusion.
